dashboard/tests/test_dashboard.py

```diff
@@ -561,5 +561,38 @@ def test_immutable_types():
         print(d3[1])
 
 
+def test_http_proxy(enable_test_module, set_http_proxy, shutdown_only):
```

Comment:
Hi @mfitton, any suggestions on this test? I can pass it on the local test. However, it still failed on CI. 
```
============================= test session starts ==============================
platform darwin -- Python 3.7.5, pytest-5.4.2, py-1.8.1, pluggy-0.13.1 -- /Users/xianyang/miniconda3/envs/torch/bin/python
cachedir: .pytest_cache
rootdir: /Users/xianyang/git/ray/python
collecting ... collected 10 items

../../python/::test_basic[ray_start_with_dashboard0] PASSED              [ 10%]
../../python/::test_nodes_update PASSED                                  [ 20%]
../../python/::test_http_get PASSED                                      [ 30%]
../../python/::test_class_method_route_table PASSED                      [ 40%]
../../python/::test_async_loop_forever PASSED                            [ 50%]
../../python/::test_dashboard_module_decorator PASSED                    [ 60%]
../../python/::test_aiohttp_cache PASSED                                 [ 70%]
../../python/::test_get_cluster_status PASSED                            [ 80%]
../../python/::test_immutable_types PASSED                               [ 90%]
../../python/::test_http_proxy PASSED                                    [100%]

=============================== warnings summary ===============================
::test_class_method_route_table
::test_class_method_route_table
  /Users/xianyang/git/ray/python/ray/tune/utils/util.py:10: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import defaultdict, deque, Mapping, Sequence

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================= 10 passed, 2 warnings in 50.62s ========================

Process finished with exit code 0

```